src/samples/flow/netctl/network-radios.fbp

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sol-fbp-runner
+
+# This file is part of the Soletta™ Project
+#
+# Copyright (C) 2015 Intel Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This program will print to console a list of all network services
+#
+# Examples:
+#  * turn on or off network connection
+#    $ ./network-radios.fbp
+
+radio(netctl/radios)
+
+_(constant/boolean:value=false) OUT -> ENABLED radio OUT -> IN _(console)
```

Comment:
here you can do a nice example with the logic:
 - `app/argv:index=1`, if there is an argument, it will convert to boolean and set `ENABLED`. As far as I remember, if no argument is given, it will send an `ERROR` packet on that port. Use that error packet as "no argument given" and if that is the case, keep monitoring. If a packet was given, then quit the application when the radio state change to the desired value.
 - `radio OUT` sent to console to show the current state.